requirements.txt

```diff
@@ -9,3 +9,4 @@ jsonschema
 PyGObject
 PyYAML
 setuptools
+varlink
```

Comment:
Should we add the dependency in `nmstate.spec` instead of here? I couldn't find any information about which one is the correct option. What do you think? @SudhanAruna @tyll

Reply:
@ffmancera It was removed nmstate.spec. The change was requested by @tyll 
https://github.com/nmstate/nmstate/pull/1081#discussion_r438329523

Reply:
The RPM dependency is generated, you can see it here for example: https://travis-ci.com/github/nmstate/nmstate/jobs/355932853#L2013
Therefore adding it only to the reqirements.txt is the right way.

Reply:
In addition, we need to generate the dependency only when installing nmstate-varlink. If not, this will break the installation for everyone.

IMO, we could remove it from requirements.txt and only add it on nmstate.spec, but I remember that you do not like this, right? @tyll